src/services/goals.js

```diff
@@ -817,15 +818,27 @@ export async function createOrUpdateGoals(goals) {
       objectives,
       status,
       createdVia,
+      isRttapa,
       ...fields
     } = goalData;
 
     // there can only be one on the goal form (multiple grants maybe, but one recipient)
     recipient = recipientId;
 
+    let isRttapaValue = null;
+
+    if (isRttapa === 'Yes') {
+      isRttapaValue = true;
+    }
+
+    if (isRttapa === 'No') {
+      isRttapaValue = false;
+    }
+
```

Comment:
What value should it use if isRttapa is not 'Yes' or 'No'?

Reply:
if it's a draft save, a user wouldn't be required to select a value, and the value might be null, so the three allowed values are true, false, and null.

Reply:
Actually, I think I'm wrong about this - it's an enum with yes and no, and allow null, so I am taking another look here

Reply:
@GarrettEHill Does this look all right to you now?